.rubocop.yml

```diff
@@ -0,0 +1,46 @@
+AllCops:
+  TargetRubyVersion: 2.5
```

Comment:
check the default version of rubocop please, and since this is a lib we should support at least the versions supported by the ruby core team in the case 2.3.0 that still receives security patches